src/simmate/toolkit/structure_prediction/evolution/workflows/fixed_composition.py

```diff
@@ -133,10 +124,15 @@ def run_config(
         #######################################################################
 
         logging.info(f"Setting up evolutionary search for {composition}")
-        # TODO: Should I add logs to inform how the workflow will be ran? Or
-        # leave it to the simmate_metadata.yaml file and database for this?
-        # logging.info(f"Individuals will be evaulated using '{subworkflow_name}'")
-        # logging.info(f"Parent selection for mutations will use '{selector_name}'")
+
+        # Convergence criteria and stop conditions can be set based on the
+        # number of atoms in the composition. Here we try to set reasons criteria
+        # for these if a user-input was not given. Note we are using max(..., N)
+        # to set an absolute minimum for these.
+        n = composition.num_atoms
+        min_structures_exact = min_structures_exact or max([int(30 * n), 100])
+        max_structures = min_structures_exact or max([int(n * 250 + n**2.75), 1500])
+        limit_best_survival = limit_best_survival or max([int(30 * n + n**2), 100])
```

Comment:
@scott-materials I'd like to have "reasonable" convergence & cutoff criteria set if the user doesn't define them. Let me know what you think of these defaults.

Reply:
@laurenmm looping lauren in too since we were chatting about this